evasquez26/angular4-startup
# S5-ONE-52 Join Course as Teacher (Missing Fix)

## Story / Task

see [ONE-52](https://bixlabs.atlassian.net/browse/ONE-52)


## What this PR does?
Finishes missing fix for U.S. ONE-52
Shows if user has already requested to join course as teacher

### Before 

Could not know if user has requested to join course as teacher

### After

Displays if user has requested to join course as teacher

![captura de pantalla de 2017-06-01 23 16 41](https://cloud.githubusercontent.com/assets/2192121/26711828/654589f8-4720-11e7-94a3-d5c21e6c8ccb.png)


### How to test / reproduce

- Login to the app
- Go to a school where you are a teacher or admin
- Go to a course from the school
- Request to join as teacher
- Reload page
- See you have a message saying you already requested to join


### TODOs

### Related PRs
N/A

## Migrations

NO


Thank you
